src/EnergyPlus/HWBaseboardRadiator.cc

```diff
@@ -1501,35 +1499,31 @@ namespace HWBaseboardRadiator {
         int SurfNum;              // Pointer to the Surface derived type
         Real64 ThisSurfIntensity; // temporary for W/m2 term for rad on a surface
 
-        int const NumHWBaseboards = state.dataHWBaseboardRad->NumHWBaseboards;
-
         // Initialize arrays
         state.dataHeatBalFanSys->SurfQHWBaseboard = 0.0;
         state.dataHeatBalFanSys->ZoneQHWBaseboardToPerson = 0.0;
 
-        for (BaseboardNum = 1; BaseboardNum <= NumHWBaseboards; ++BaseboardNum) {
-            auto &HWBaseboard = state.dataHWBaseboardRad->HWBaseboard(BaseboardNum);
+        for (BaseboardNum = 1; BaseboardNum <= state.dataHWBaseboardRad->NumHWBaseboards; ++BaseboardNum) {
```

Comment:
Learning opportunity. I searched/highlighted, on this page, for BaseboardNum. The only occurrence is at line 1508. So then this could be:

    for (auto &thisHWBB : state.dataHWBaseboardRad->HWBaseboard)